src/config.rs

```diff
@@ -221,6 +223,56 @@ impl From<Bitrate> for LSBitrate {
     }
 }
 
+/// Libspotify audio formats
+static AUDIO_FORMAT_VALUES: &[&str] = &["F32", "S32", "S24", "S24_3", "S16"];
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, StructOpt)]
```

Comment:
> Also not much used to github PR, how the hell is there no comment thread ? Or am I missing something ?

You should in theory be able to reply inside this thread or did I misread your question?

> In the derive block, at least Deserialize is necessary. Otherwise I really don't have an opinion on this :)

Oh, right. We'll see what maintainers think, but I'd just leave it there for now.